tests/scripts/run_pre_post_commit_regressions_multi_device.sh

```diff
@@ -41,9 +41,13 @@ pytest models/demos/falcon40b/tests/test_falcon_end_to_end.py::test_FalconCausal
 
 # Falcon40B 8 chip prefill tests; we need 8x8 grid size for prefill tests on wormhole_b0
 if [ "$ARCH_NAME" == "wormhole_b0" ]; then
+  # BFP8
   WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/falcon40b/tests/test_falcon_end_to_end.py::test_FalconCausalLM_end_to_end_with_program_cache[BFLOAT8_B-DRAM-falcon_40b-layers_1-prefill_seq32-8chips-disable_program_cache]
   WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/falcon40b/tests/test_falcon_end_to_end.py::test_FalconCausalLM_end_to_end_with_program_cache[BFLOAT8_B-DRAM-falcon_40b-layers_1-prefill_seq128-8chips-disable_program_cache]
   WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/falcon40b/tests/test_falcon_end_to_end.py::test_FalconCausalLM_end_to_end_with_program_cache[BFLOAT8_B-DRAM-falcon_40b-layers_1-prefill_seq2048-8chips-disable_program_cache]
+  # BFP16
+  WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/falcon40b/tests/test_falcon_end_to_end.py::test_FalconCausalLM_end_to_end_with_program_cache[BFLOAT16-DRAM-falcon_40b-layers_1-prefill_seq128-8chips-disable_program_cache]
```

Comment:
Can you please target a file only, not a specific file?

we are trying to cleanup test invocations.

Reply:
You can use skips, or refactor the single test into a single file etc

Reply:
We have a lot of tests in there that we locally run but don't want to run through CI (e.g. more layers). What's the reason for targeting files rather than specific tests?

Reply:
You can do `@pytest.skip(reason="...")`. We can have `skip_on_ci` just do the same thing, but I dont see a point?

Reply:
@TT-BrianLiu there's more discussion in #7598

Reply:
@TT-BrianLiu tl;dr because there are tests people want to be able to run, but ignore on CI